app/components/contributors/edit_component.html.erb

```diff
@@ -31,7 +30,11 @@
     <div class="form-check">
       <%= form.radio_button :with_orcid, true, class: 'form-check-input' %>
       <%= form.label :with_orcid_true, 'ORCID iD', class: 'form-check-label' %>
-      <%= render Elements::Forms::TextFieldComponent.new(form:, field_name: :orcid, hidden_label: true) %>
+      <%= tag.div class: orcid_container_classes do %>
```

Comment:
Inline the text field component because we need to inject the form-group button conditionally.